templates/group.html

```diff
@@ -1 +1 @@
-<a ng-href="#/{{ctrl.group}}">{{ctrl.group}}</a>
+<a ng-href="#/{{ctrl.group}}">{{ctrl.group}} ({{ctrl.groupCount}})</a>
```

Comment:
Please format menus according to guidelines: https://docs.nextcloud.com/server/11/developer_manual/app/css.html#menus